code/modules/mob/living/carbon/human/emote.dm

```diff
@@ -530,8 +530,8 @@
 						for(var/mob/living/M in range(0))
 							if(M != src)
 								visible_message("\red <b>[src]</b>'s ass blasts <b>[M]</b> in the face!", "\red You ass blast <b>[M]</b>!")
-								M.apply_damage(75,"brute","head")
-								add_logs(src, M, "superfarted on", object=null, addition=" (DAMAGE DEALT: 75)")
+								M.apply_damage(40,"brute","head")
```

Comment:
What if we checked whether the target and src are on the ground? it seems odd if someone is crit lying on the ground and they're able to superfart on the head of someone who's standing up